roles/manage_dbserver/tasks/pgpass_line.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+- name: Check if pgpass exists or not
+  stat:
+    path: "{{ pgpass_file| default('~/.pgpass') }}"
+  become: yes
+  become_user: "{{ pg_owner }}"
+  register: pgpass_file_exists
+  no_log: "{{ disable_logging }}"
+
+- name: Prepare regex and entry line
+  set_fact:
+      search_line: "^{{ line_item.host|default('*') }}:{{ pg_port }}:{{ line_item.database|default('*') }}:{{ line_item.user }}"
+      pgpass_line: "{{ line_item.host|default('*') }}:{{ pg_port }}:{{ line_item.database|default('*') }}:{{ line_item.user }}:{{ line_item.password }}"
+  when:
+    - pg_pgpass_values|length > 0
+
+- name: delete entries in .pgpass
+  lineinfile:
+    path: "{{ pgpass_file | default('~/.pgpass') }}"
+    regexp: "{{ search_line|replace('*','\\*') }}"
```

Comment:
I'd recommend using `"{{ search_line | regex_escape() }}"`

Reply:
Thank you for the tip! Fixed!